sdks/java/io/google-cloud-platform/expansion-service/build.gradle

```diff
@@ -36,6 +36,9 @@ dependencies {
     permitUnusedDeclared project(":sdks:java:io:google-cloud-platform") // BEAM-11761
     implementation project(":sdks:java:extensions:schemaio-expansion-service")
     permitUnusedDeclared project(":sdks:java:extensions:schemaio-expansion-service") // BEAM-11761
+    implementation project(":sdks:java:managed")
+    permitUnusedDeclared project(":sdks:java:managed") // BEAM-11761
```

Comment:
Notes, no action required on this PR:

 - This is a link to Jira, so probably there's a github issue it is migrated to
 - This should be equivalent to `runtimeOnly` because it is "implementation" but no static references to it. I would guess this works the same, or else the uberjar plugin might not treat it right.
 - Putting these deps into a docker container without making an uber jar would honestly be better in the case where it _does_ end up in a container, so we keep the original jar metadata.